style.css

```diff
@@ -0,0 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;900&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  background-position: 20px 10px;
+  background-image: url(./header-shapes\ mobile.png);
```

Comment:
- [ ] Background image should be aligned as in the original website, Please fix it.
- Original
![top2](https://user-images.githubusercontent.com/26767383/149210560-7037b188-b33d-494b-b2b5-1d055f16b17b.PNG)
- your website
 
![top1](https://user-images.githubusercontent.com/26767383/149210642-013c1eb2-a185-47f5-9d7d-223268b6b59c.PNG)